### PR TITLE
[DOCS] Updates to outlier detection release highlight

### PR DESCRIPTION
**Pull request by szabosteve**
This PR changes the description of outlier detection in the release highlights for 7.3.
https://www.elastic.co/guide/en/elasticsearch/reference/7.3/release-highlights-7.3.0.html#_outlier_detection

### CODE REVIEW

File: docs/reference/release-notes/highlights-7.3.0.asciidoc [hunk 1]
```diff
@@ -144,16 +144,31 @@ NOTE: Data frames are only available with the default distribution of {es}.
...
-from those of the clustered data point. An outlier score can be used to indicate
-how different an entity is from other entities in the index based on the
-dimensions that you supply.
-
-NOTE: Outlier detection requires a platinum license.
+[discrete]
+[[release-highlights-7.3.0-outlier-detection]]
+==== Discover your most unusual data using {oldetection}
+
+The goal of {stack-ov}/security-privileges.html[{oldetection}] is to find
```
Comment (lcawl) [inline docs/reference/release-notes/highlights-7.3.0.asciidoc]:
```suggestion
The goal of {stack-ov}/dfa-outlier-detection.html[{oldetection}] is to find 
```

File: docs/reference/release-notes/highlights-7.3.0.asciidoc [hunk 2]
```diff
@@ -144,16 +144,31 @@ NOTE: Data frames are only available with the default distribution of {es}.
...
+We use unsupervised {oldetection} which means there is no need to provide a 
+training data set to teach {oldetection} to recognize outliers. In practice, 
+this is achieved by using an ensemble of distance based and density based 
+techniques to identify those data points which are the most different from the 
+bulk of the data in the index. We assign to each analysed data point an 
+{olscore}, which captures how different the entity is from other entities in the 
+index.
+ 
+In addition to new {oldetection} functionality, we are introducing the 
+Evaluate API, which allows users to compute a range of performance metrics such
```
Comment (lcawl) [inline docs/reference/release-notes/highlights-7.3.0.asciidoc]:
```suggestion
evaluate {dfanalytics} API, which enables you to compute a range of performance metrics such 
```

File: docs/reference/release-notes/highlights-7.3.0.asciidoc (hunk 1 above)
Comment (szabosteve) [reply docs/reference/release-notes/highlights-7.3.0.asciidoc]: Thanks!

File: docs/reference/release-notes/highlights-7.3.0.asciidoc [hunk 3]
```diff
@@ -144,16 +144,31 @@ NOTE: Data frames are only available with the default distribution of {es}.
...
+index.
+ 
+In addition to new {oldetection} functionality, we are introducing the 
+Evaluate API, which allows users to compute a range of performance metrics such 
+as confusion matrices, precision, recall, the 
+https://en.wikipedia.org/wiki/Receiver_operating_characteristic[receiver-operating characteristics (ROC) curve] 
+and the area under the ROC curve. If you are running {oldetection} on a source 
+index that has already been labeled to indicate which points are truly outliers 
+and which are normal, you can use the 
+{ref}/evaluate-dfanalytics.html[Evaluate API] to assess the performance of the
```
Comment (lcawl) [inline docs/reference/release-notes/highlights-7.3.0.asciidoc]: I suggest putting the link on the first occurrence instead. 
```suggestion
evaluate {dfanalytics} API to assess the performance of the 
```

**Review by lcawl — APPROVED** [review]
LGTM and builds successfully

**Pull request merged**